gradio/utils.py

```diff
@@ -279,6 +279,8 @@ def delete_none(_dict):
     """
     if isinstance(_dict, dict):
         for key, value in list(_dict.items()):
+            if key == "value":
```

Comment:
Suggest adding something in the docstring to mention the fact that we exclude keys whose name is "value" and the rationale (in case someone is reading the code later wonders why)